### PR TITLE
GEODE-9006: fix flaky native Redis memory usage test

### PR DESCRIPTION
**Pull request by jhutchison**
- move test to only be run against apis-compatable-with-redis
- rename test be more specific

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractRedisMemoryStatsIntegrationTest.java [hunk 1]
```diff
@@ -33,10 +33,6 @@
 public abstract class AbstractRedisMemoryStatsIntegrationTest implements RedisIntegrationTest {
```
Comment (sabbey37) [inline geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractRedisMemoryStatsIntegrationTest.java:34]: Should we remove this class altogether along with the `MemoryStatsNativeRedisAcceptanceTest` class and just have the Radish test class?

Comment (jhutchison) [reply geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractRedisMemoryStatsIntegrationTest.java:31]: changes made as per convo at retro (data load in test increased)

File: geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/server/MemoryStatsNativeRedisAcceptanceTest.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ public int getPort() {
 
   @Override
   public void configureMemoryAndEvictionPolicy(Jedis jedis) {
-    jedis.configSet("maxmemory", "2000000");
+    jedis.configSet("maxmemory", "18000000");
```
Comment (sabbey37) [inline geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/server/MemoryStatsNativeRedisAcceptanceTest.java:34]: Is there any reason we chose 18,000,000?  Locally it passes with 10,000,000. I guess it doesn't really matter though as long as there's no negative impact to having a larger max memory.

**Review by sabbey37 — APPROVED** [review]
Thanks for fixing this!

File: geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/server/MemoryStatsNativeRedisAcceptanceTest.java (hunk 2 above)
Comment (jhutchison) [reply geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/server/MemoryStatsNativeRedisAcceptanceTest.java:34]: I tried to match it to the size of the data that's used, but I guess I was a little off.  I can decrease.

**Pull request merged**